Recommend WSL2 only and add disclaimer (#2108)
dfinity/portal: docs/developer-docs/setup/install/index.mdx

The **IC SDK** is natively supported on Linux or macOS 12.\* Monterey or later. 

> There is no native support for <code>dfx</code> on Windows. However, by installing the Windows Subsystem for Linux (WSL 2), you can run dfx also on a Windows system as described below (not all features may be supported though).

Reviewer: ```suggestion
There is no native support for <code>dfx</code> on Windows. However, by installing the Windows Subsystem for Linux (WSL 2), you can run dfx on a Windows system as described below.

:::caution
Not all features of dfx may be supported on WSL 2.
:::
```
